pyspark/bigdl/models/rnn/rnnexample.py

```diff
@@ -117,11 +117,34 @@ def padding(features, label, length):
     return sample_rdd, val_sample_rdd, total_vocab_len
 
 def build_model(input_size, hidden_size, output_size):
-    model = Sequential()
-    model.add(Recurrent()
-              .add(RnnCell(input_size, hidden_size, Tanh())))\
-        .add(TimeDistributed(Linear(hidden_size, output_size)))
-    model.reset()
+    # Sequential can be used for MklBlas only. To use MklDnn backend,
+    # you should define the model with Model(graph container)
+    if get_bigdl_engine_type() == "MklBlas":
+        model = Sequential()
+        model.add(Recurrent()
+                  .add(RnnCell(input_size, hidden_size, Tanh())))\
+            .add(TimeDistributed(Linear(hidden_size, output_size)))
+        model.reset()
+
+    # In order to use MklDnn as the backend, you should:
+    # 1. Define a model with Model(graph container)
+    # 2. Specify the input and output formats of it.
+    #    BigDL needs these format information to build IRGraph from StaticGraph for MklDnn computing
+    # 3. Running spark-submit command with correct configurations
+    #    --conf "spark.driver.extraJavaOptions=-Dbigdl.engineType=mkldnn"
+    #    --conf "spark.executor.extraJavaOptions=-Dbigdl.engineType=mkldnn"
+    # LSTM supports MklDnn backend. Simple RNN does not for now.
+    else:
+        input1 = Input()
```

Comment:
same, you don't need to define model again.

Reply:
Probably have to, since the original example uses a simple rnn but we do not have mkldnn support for this. An lstm model is used for mkldnn case.